package.xml

```diff
@@ -2,27 +2,27 @@
   <name>octomap_msgs</name>
   <version>0.3.3</version>
   <description>
-   This package provides messages and serializations / conversion for the <a href="http://octomap.github.com">OctoMap library</a>.    
+   This package provides messages and serializations / conversion for the <a href="http://octomap.github.com">OctoMap library</a>.
   </description>
 
   <author email="armin@hornung.io">Armin Hornung</author>
   <maintainer email="armin@hornung.io">Armin Hornung</maintainer>
 
-  <license>BSD</license>  
+  <license>BSD</license>
 
   <url type="website">http://ros.org/wiki/octomap_msgs</url>
   <url type="bugtracker">https://github.com/OctoMap/octomap_msgs/issues</url>
 
-  <build_depend>ament_cmake</build_depend>
-  <build_depend>message_generation</build_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
+
   <build_depend>rosidl_default_generators</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
 
-  <exec_depend>message_generation</exec_depend> <!-- provide message generation to downstream packages -->
-  <exec_depend>message_runtime</exec_depend>
-  <exec_depend>std_msgs</exec_depend>
```

Comment:
Why remove the ``<exec_depend>``?

Reply:
`exec_depend` is redundant with `depend`. In general if a dependency is both an `exec_depend` and a `build_depend` I prefer to use `depend` to keep the code more compact